.github/workflows/lint-collection.yml

```diff
@@ -41,6 +41,8 @@ jobs:
     steps:
       # Checks-out the repository under $GITHUB_WORKSPACE, so it's accessible to the job
       - uses: actions/checkout@v3
+        with:
+          path: stackhpc-github
```

Comment:
This will be the collection repo, not .github. You might need another checkout step

Reply:
Good point. Thing is I'm not sure there is a context variable containing the git_ref for this reusable workflow i.e need to get main from: 

```
  lint:
    uses: stackhpc/.github/.github/workflows/lint-collection.yml@main
```
Wonder whether its easier just to include it inline in the workflow?

Reply:
heh, this should have been easy. Might be simplest to just checkout main of .github

Reply:
Nightmare in 2.12:

```
This version of ansible-test cannot be executed with Python version 3.11.4. Supported Python versions are: 3.8, 3.9, 3.10
```

https://github.com/stackhpc/ansible-collection-linux/actions/runs/5657449401/job/15326522093?pr=11#step:6:25